.gitignore

```diff
@@ -24,6 +24,10 @@ test-output/
 *.o
 *.so
 
+# TypeScript built source #
+###########################
+dashboard/src/**/*.js
```

Comment:
Sorry. It needed for some built in compiler. It very useful to go through to resources. I have removed this commit.